kong/api/routes/plugins.lua

```diff
@@ -13,12 +13,15 @@ local setmetatable = setmetatable
 
 
 local get_plugin = endpoints.get_entity_endpoint(kong.db.plugins.schema)
-local put_plugin = endpoints.put_entity_endpoint(kong.db.plugins.schema)
 local delete_plugin = endpoints.delete_entity_endpoint(kong.db.plugins.schema)
 
 
 local function before_plugin_for_entity(entity_name, plugin_field)
   return function(self, db, helpers)
+    if ngx.req.get_method() == "PUT" then
```

Comment:
`kong.request.get_method`? :)

Reply:
`self.req.method`?

Reply:
The Admin API code doesn't use `kong.request.get_method()` anywhere yet, and uses more `ngx.req.get_method()` than it uses `self.req.method` (which is actually not documented anywhere I could find in Lapis, hence my aversion against using it).